integration-tests/integration/whereabouts/selectResidentialLocation.spec.js

```diff
@@ -0,0 +1,55 @@
+const moment = require('moment')
+
+// copy of shared/getCurrentTimePeriod.js
```

Comment:
Didn't want to include it directly incase it changes and the tests pass incorrectly.